TestTicTacToe/BoardTests.cs

```diff
@@ -0,0 +1,65 @@
+using TicTacToe;
+using Xunit;
+
+namespace TestTicTacToe
+{
+    public class BoardTests
+    {
+        [Fact]
+        public void Should_Place_Marker_In_The_Correct_Position_If_Empty_Cell()
+        {
+            // Arrange 
+            var board = new Board();
+            var move = new Move(1 - Constants.IndexingAdjustment, 1 - Constants.IndexingAdjustment);
+            
+            var expectedBoard = new[,]
+            {
+                {1, 0, 0},
+                {0, 0, 0},
+                {0, 0, 0}
+            };
+        
+            // Act
+            board.PlaceMarker(move, 1);
+            var actualBoard = board.GetBoard();
```

Comment:
I don't think this variable actualBoard is providing much value so in terms of keeping things small, I would just put board.GetBoard() in the Assert directly.